src/helper/UserSession.cpp

```diff
@@ -50,7 +50,12 @@ namespace SDDM {
         if (env.value(QStringLiteral("XDG_SESSION_CLASS")) == QLatin1String("greeter")) {
             QProcess::start(m_path);
         } else if (env.value(QStringLiteral("XDG_SESSION_TYPE")) == QLatin1String("x11")) {
-            const QString cmd = QStringLiteral("%1 \"%2\"").arg(mainConfig.X11.SessionCommand.get()).arg(m_path);
+#ifdef Q_OS_FREEBSD
+            const QString cmdFormat = QStringLiteral("%1 %2");
+#else
+            const QString cmdFormat = QStringLiteral("%1 \"%2\"");
+#endif
+            const QString cmd = cmdFormat.arg(mainConfig.X11.SessionCommand.get()).arg(m_path);
             qInfo() << "Starting:" << cmd;
             QProcess::start(cmd);
```

Comment:
Should be ported to:
`QProcess::start(mainConfig.X11.SessionCommand.get(), {m_path});`

The current QProcess::start(cmd) that we're using everywhere is deprecated anyway. I'm not sure if SessionCommand can have arguments, if so we'll need to use QProcess::splitCommand().